.github/workflows/validate.yaml

```diff
@@ -30,6 +30,12 @@ jobs:
         run: |
           go install github.com/mikefarah/yq/v3@latest
 
+      - name: Install OPM
+        run: |
+          wget https://github.com/operator-framework/operator-registry/releases/download/v1.47.0/linux-amd64-opm && \
+            chmod +x linux-amd64-opm && \
+            mv linux-amd64-opm /usr/local/bin/opm
```

Comment:
alternatively you can also use `go install`, so you don't have to care about arch & path:
`go install github.com/operator-framework/operator-registry/cmd/opm@v1.47.0`